EIPS/eip-5320.md

```diff
@@ -0,0 +1,156 @@
+---
+eip: 5320
+title: Harberger Tax NFT
+description: Interface for Harberger Tax compatible NFTs.
+author: Green (@greenlucid)
+discussions-to: https://ethereum-magicians.org/t/eip-5320-harberger-taxes-nft/10084
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-23
+requires: 20, 721
+---
+
+## Abstract
+
+The following standard allows for implementation of a standard API for NFTs subject to Harberger Taxes within smart contracts. Harberger Taxes work in the following way, admitting some variants:
+
+1. Alice owns an Item, and values at an amount X. 
+2. Alice pays taxes according to a tax rate R to a collector, depending on this self-assesed value.
+3. Bob offers Alice an amount Y. Because Y is higher than X, Alice is forced to sell the Item to him.
+4. Bob self-asseses the value too at Z, and now pays taxes over it.
+
+This mechanic sets a stage in which actors won't overvalue the item to avoid paying excessive taxes, but won't undervalue the item either, to avoid a third party to force them to sell it.
+
+The main obvious case of Harberger Taxes is to be used as a coordination mechanism for land allocation, but, to give some examples:
+
+- Real-world land allocation.
+- Digital land allocation (videogames, social experiments...).
+- Advertisement Space in websites.
+- Governance mechanisms that use "seats" subject to Harberger.
+- Social media that requires holding a token to interact.
+
```

Comment:
These use cases can go in the Motivation section. They don't belong in a technical abstract.